### ISSUE TITLE
Cleanup CSS

### ISSUE THREAD

**Issue by jonfroehlich** [makeabilitylab/makeabilitylabwebsite#294]
CSS needs to be massively refactored and cleaned up.

I'm probably best suited to do this since I recently led the redesign of the site. I want to:

- drastically decrease the total number of individual classes. there are many classes that should be, at a high level, shared. So, rather than like talk-thumbnail and project-thumbnail, we should have artifact-thumbnail as a general class... and then if this needs to be overloaded by an individual page (e.g., talks), then we can do that overload in the talks.css

- decrease number of individual css files

- possibly rename styles.css to base.css to better fit within django framework (however, one reason not to do this is that it seems web devs often use styles.css to mean the main CSS file)...

**Comment by jonfroehlich** [comment]
Ideally, this should be done before our hackathon on Monday, Dec 18

**Comment by jonfroehlich** [comment]
Closing in favor of https://github.com/makeabilitylab/makeabilitylabwebsite/issues/664
